clients/include/testing_trmm_batched.hpp

```diff
@@ -163,6 +148,66 @@ hipblasStatus_t testing_trmm_batched(const Arguments& argus)
         {
             unit_check_general<T>(M, N, batch_count, ldb, hB_array, hB_copy_array);
         }
+        if(argus.norm_check)
+        {
+            hipblas_error
+                = norm_check_general<T>('F', M, N, ldb, hB_copy_array, hB_array, batch_count);
+        }
+    }
+
+    if(argus.timing)
+    {
+        hipStream_t stream;
+        status = hipblasGetStream(handle, &stream);
+        if(status != HIPBLAS_STATUS_SUCCESS)
+        {
+            hipblasDestroy(handle);
+            return status;
+        }
+
+        int runs = argus.cold_iters + argus.iters;
+        for(int iter = 0; iter < runs; iter++)
+        {
+            if(iter == argus.cold_iters)
+                gpu_time_used = get_time_us_sync(stream);
+
+            status = hipblasTrmmBatchedFn(handle,
+                                          side,
+                                          uplo,
+                                          transA,
+                                          diag,
+                                          M,
+                                          N,
+                                          &alpha,
+                                          dA_array,
+                                          lda,
+                                          dB_array,
+                                          ldb,
+                                          batch_count);
+
+            if(status != HIPBLAS_STATUS_SUCCESS)
+            {
+                hipblasDestroy(handle);
+                return status;
+            }
+        }
+        gpu_time_used = get_time_us_sync(stream) - gpu_time_used;
+
+        ArgumentModel<e_side_option,
+                      e_uplo_option,
+                      e_transA_option,
+                      e_diag_option,
+                      e_M,
+                      e_N,
+                      e_lda,
+                      e_ldb,
+                      e_batch_count>{}
+            .log_args<T>(std::cout,
+                         argus,
+                         gpu_time_used,
+                         trmm_gflop_count<T>(M, N, K),
+                         trmm_gbyte_count<T>(M, N, K),
+                         hipblas_error);
```

Comment:
see comment above, should be

    trmm_gflop_count<T>(M,N,side)
    trmm_gbyte_count<T>(M,N,side)